src/main/kotlin/no/nav/syfo/personstatus/infrastructure/database/queries/getFromPersonOversiktStatus.kt

```diff
@@ -17,6 +17,7 @@ const val queryGetPersonOversiktStatusList =
     WHERE fnr = ?
     """
 
+// Deprecated: Use getPersonOversiktStatus in PersonOversiktStatusRepository instead
```

Comment:
Hva med følgende i stedet?
```suggestion
@Deprecated("Use getPersonOversiktStatus in PersonOversiktStatusRepository instead")
```

Reply:
Mye bedre! 🤌